subprojects/core/src/integTest/groovy/org/gradle/caching/internal/FinalizeBuildCacheConfigurationBuildOperationIntegrationTest.groovy

```diff
@@ -89,9 +89,9 @@ class FinalizeBuildCacheConfigurationBuildOperationIntegrationTest extends Abstr
         then:
         def result = result()
 
-        !result.disabled
-        !result.localDisabled
-        !result.remoteDisabled
+        result.enabled
+        result.localEnabled
+        !result.remoteEnabled
```

Comment:
The assertion was wrong in the first place. If the DSL is not defined for remote, `remoteEnabled` should be false.